Sources/App/configure.swift

```diff
@@ -69,4 +72,8 @@ public func configure(_ config: inout Config, _ env: inout Environment, _ servic
     var middlewares = MiddlewareConfig()
     middlewares.use(ErrorMiddleware.self)
     services.register(middlewares)
+
+    // Some services (like JIRA) might need a slower client which handles rate-limiting APIs and quotas
+    let slowClient = SlowClient()
```

Comment:
Rate-Limiting-aware client which sends requests one after the other instead of in parallel. Since the current theory for the failure is us sending too many requests at once for the JIRA API to handle, the hope here is to only send JIRA requests sequentially instead to limit the risk and see if it solves the issue on our next invocation of `/crp` during next release.